docs/ingress-service-dns-with-coredns.md

```diff
@@ -34,6 +34,8 @@ Install Federation-v2 with minikube in [User Guide](https://github.com/kubernete
 Install ExternalDNS with CoreDNS as backend in your host cluster. You can follow the [tutorial](https://github.com/kubernetes-incubator/external-dns/blob/master/docs/tutorials/coredns.md).
 
 To make it work for federation resources, you need to use below ExternalDNS deployment instead of the one in the tutorial.
+**Note**: You should replace parameter of `parameters: example.org` with `parameters: example.com` when [Installing CoreDNS](https://github.com/kubernetes-incubator/external-dns/blob/master/docs/tutorials/coredns.md#installing-coredns)
```

Comment:
Can you please move this before Line 36?

/cc @xunpan

Reply:
I have tried that: delete the empty line-35, but the output looks ugly.

Reply:
I do not mean delete line 35, but update the document as follows:

Install ExternalDNS with CoreDNS as backend in your host cluster. You can follow the [tutorial](https://github.com/kubernetes-incubator/external-dns/blob/master/docs/tutorials/coredns.md).

**Note**: You should replace parameter of `parameters: example.org` with `parameters: example.com` when [Installing CoreDNS](https://github.com/kubernetes-incubator/external-dns/blob/master/docs/tutorials/coredns.md#installing-coredns)

To make it work for federation resources, you need to use below ExternalDNS deployment instead of the one in the tutorial.

**Note**: You should replace value of `ETCD_URLS` with your own etcd client service IP address.

Reply:
Copy, do it now